examples/variational_autoencoder_deconv.py

```diff
@@ -13,6 +13,8 @@
 from keras import objectives
 from keras.datasets import mnist
 
+K.set_image_dim_ordering('th') # this is a Theano oriented example
```

Comment:
It seems you have included with this PR code that shouldn't be there (or may it's just this one line)? I made this example compatible with both dim orderings a while ago.